numpy/linalg/linalg.py

```diff
@@ -2556,11 +2561,15 @@ def norm(x, ord=None, axis=None, keepdims=False):
             # special case for speedup
             s = (x.conj() * x).real
             return sqrt(add.reduce(s, axis=axis, keepdims=keepdims))
+        elif ord in ('f', 'fro'):
+            raise ValueError("Cannot compute Frobenius norm for vectors, "
+                             "vectors only support integer norm orders. For "
+                             "example for the L2 norm use ord=2 instead")
+        elif ord is 'nuc':
+            raise ValueError("Nuclear norm is not supported for vectors")
```

Comment:
Since none of the named kwargs ('fro', 'nuc') are valid for vectors, it seems to me these checks could be consolidated.
```suggestion
        elif ord in ('f', 'fro', 'nuc'):
            raise ValueError(f"Invalid norm order '{ord}' for vectors")
```